packages/neuron-ui/src/components/Router/index.tsx

```diff
@@ -23,75 +25,79 @@ interface CustomRoute {
   path: string
   name: string
   icon?: Icon
-  showInSidebar?: boolean
   exact?: boolean
   component: React.ComponentType
 }
 
 export const routes: CustomRoute[] = [
   {
     name: 'Header',
-    showInSidebar: false,
```

Comment:
@Keith-CY I removed this property because we might use part of routes in several places, not just on the sidebar. Considering that, the full routes collection should not have concern about where they'll be used. (Otherwise it'll have `showInSidebar`, `showInTopNav`, `showInFooter`, etc. and quickly become too complicated.)